filip/semantics/semantics_models.py

```diff
@@ -240,9 +227,6 @@ def get_all_field_names(self, field_name: Optional[str] = None) \
         """
         pass
 
-    class Config:
```

Comment:
not necessary anymore?

Reply:
I dont know. I think you remove it [here](https://github.com/RWTH-EBC/FiLiP/pull/199/commits/ff9b17a8385e406a9deda0662b88b61643e605d4)

Reply:
Cant read you link

Reply:
I link a commit of yours. If you check the filip/semantics/semantics_models.py in that commit, you can see the code has been removed there :)